File: mobile/src/interface/indicator.ts

interface IIndicator{
    id: number,
    name: string,
    sc: any,
    kpi: boolean,
    national: any,
    state: any,
    district: any,
    block: any,
    unit: any,
    highisgood: any,
    recSector: ISector,
    subgroup: any,
    slugidindicator: number,
    createdDate:string,
    lastModified:string
    nucolor:boolean,
    nitiaayog:boolean,
    checked?:boolean,
    thematicKpi:boolean,
    ssv:boolean,
    hmis:boolean,
    thematicNational:any,
    thematicState:any,
    thematicDistrict:any,
    indicator?:number,
    sectors: ISector[]

}

File: backend/src/main/java/in/co/sdrc/cap/mobile/model/Indicator.java
package in.co.sdrc.cap.mobile.model;


import java.util.List;

import lombok.Data;

@Data
public class Indicator {
	
	private int id;
	private String name;
	private Sector recSector;
	private Unit unit;
	private Subgroup subgroup;
	private List<Sector> sectors;
}
